modules/markComplete.js

```diff
@@ -0,0 +1,12 @@
+import updateStatus from './updateStatus.js';
+
+export default function markComplete(event, toDoList) {
+  const checkbox = event.target;
```

Comment:
- [ ]  Kindly check my comment on the use of es6 on the addToList file which is very detailed 👍